File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
    let answer = 0;
    let n=0;
    for(let i=0; i<num_list.length;i++){
        n=num_list[i];
        while(true){
            if(n==1) break;
            if(n%2==0) {
                n=n/2;
                answer++;
            }
            else {
                n=(n-1)/2;
                answer++;
            }
        }
    }
    return answer;
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.py
def solution(num_list):
    answer = 0
    n=0
    for i in range(len(num_list)):
        n=num_list[i]
        while True:
            if n==1: break
            if n%2==0:
                n=int(n/2)
                answer+=1
            else:
                n=int((n-1)/2)
                answer+=1
    return answer